docs/monorepo.md

```diff
@@ -1,15 +1,20 @@
 # Working with the Monorepo
 
-Calypso is a monorepo. In addition to the Calypso application, it also hosts a number of independent modules that are published to NPM.
+Calypso is a monorepo. In addition to the Calypso application, it also hosts a number of independent packages that are published to NPM.
 
 ## Module Layout
 
-These modules live under the `packages` and `apps` directories, one folder per module.
+These packages live under the `packages` and `apps` directories, one folder per module.
 
-Two different directories for modules are:
+Two different directories with packages are:
 
-`/packages` — projects and libraries that we might publish as [NPM packages](https://docs.npmjs.com/about-packages-and-modules). Typically used also elsewhere in Calypso and build on `yarn start`. See "Publishing" below.
-`/apps` — projects that can produce independent, binary-like outputs deployed elsewhere. Typically not published to NPM or build on `yarn start`.
+`/packages` — projects and libraries that we might publish as [NPM packages](https://docs.npmjs.com/about-packages-and-modules). Typically used also elsewhere in Calypso. See "Publishing" below.
+`/apps` — WPCOM plugins
+`/desktop` - WP Desktop app. Currently this is not part of the monorepo in the sense that it has its own dependency tree (its own `yarn.lock`)
```

Comment:
We could describe the reason: the `desktop/` folder needs its own isolated `node_modules` folder with the Electron stuff, because the folder will be bundled into the resulting app.